app/services/import_kcsc.rb

```diff
@@ -0,0 +1,49 @@
+class ImportKCSC
+  def self.with(http = HTTParty, model_klass = Organisation)
```

Comment:
is the idea to change the `model_klass` to `Service` after it's ready?

Reply:
not necessarily - the main reason to have the model_klass is dependency injection which provides flexibility that could be used in that way, but I have it there to isolate the tests from ActiveRecord and the DB.

once we have services in place I'm thinking we probably create both and organisation and a service to correspond to each item coming back from the API, since the API is combining elements of both on the other side.  However that's just my current intuition ...